certora/specs/AccessControlDefaultAdminRules.spec

```diff
@@ -0,0 +1,379 @@
+import "helpers.spec"
+import "methods/IAccessControlDefaultAdminRules.spec"
+import "methods/IAccessControl.spec"
+import "AccessControl.spec"
+
+use rule onlyGrantCanGrant filtered {
+  f -> f.selector != acceptDefaultAdminTransfer().selector
+}
+
+// use rule revokeRoleEffect filtered {
+//   role -> role != DEFAULT_ADMIN_ROLE()
+// }
+
+// use rule renounceRoleEffect filtered {
+//   role -> role != DEFAULT_ADMIN_ROLE()
+// }
+
+/*
+┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ Helpers                                                                                                             │
+└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+*/
+
+function max_uint48() returns mathint {
```

Comment:
Is there a difference between `function` and `definition`?

Reply:
Now that you mention it, I don't think so. At least for these particular examples.
I'd keep it like this for consistency at the moment but we can do a PR after changing these kind of functions for definitions.